hacks/hacks.mozilla.org.1.md

```diff
@@ -0,0 +1,118 @@
+# Dweb: Matrix
+
+## Author info: Full name, photo of yourself, and 1-2 sentence bio
+
+## What is Matrix?
+
+Matrix is an open standard for *interoperable*, *decentralised*, *real-time* communication over IP. It provides a standard HTTP API for publishing and subscribing to real-time data in specified channels, which means it can be used to power Instant Messaging, VoIP/WebRTC signalling, Internet of Things communication, and anything else that could be transmitted over HTTP. The most common use of Matrix today is as an Instant Messaging platform.
+
+* Matrix is *interoperable* in that it can freely communicate with other platforms. Matrix messages are JSON, and easy to parse. Bridges are provided to enable communication with other platforms.
```

Comment:
interoperable in that it follows an [open standard](https://matrix.org/docs/spec) and can freely...